extensions/librdkafka/PublishKafka.cpp

```diff
@@ -82,10 +82,33 @@ core::Property PublishKafka::TargetBatchPayloadSize(
     core::PropertyBuilder::createProperty("Target Batch Payload Size")->withDescription("The target total payload size for a batch. 0 B means unlimited (Batch Size is still applied).")
         ->isRequired(false)->withDefaultValue<core::DataSizeValue>("512 KB")->build());
 core::Property PublishKafka::AttributeNameRegex("Attributes to Send as Headers", "Any attribute whose name matches the regex will be added to the Kafka messages as a Header", "");
-core::Property PublishKafka::QueueBufferMaxTime("Queue Buffering Max Time", "Delay to wait for messages in the producer queue to accumulate before constructing message batches", "");
-core::Property PublishKafka::QueueBufferMaxSize("Queue Max Buffer Size", "Maximum total message size sum allowed on the producer queue", "");
-core::Property PublishKafka::QueueBufferMaxMessage("Queue Max Message", "Maximum number of messages allowed on the producer queue", "");
-core::Property PublishKafka::CompressCodec("Compress Codec", "compression codec to use for compressing message sets", COMPRESSION_CODEC_NONE);
+
+const core::Property PublishKafka::QueueBufferMaxTime(
```

Comment:
to nitpick: with method chaining I don't know what the style guideline is, but I prefer the operator on the next line (and it seems that other properties do it like this, in this file)

Reply:
+1.

That's the way linter accepts it as well, so please change as @adamdebreceni suggest!

Reply:
I like it this way, so 2/2 :)

Reply:
Not sure if this still applies after your linter changes, but previously the linter only accepted the "->" operators to be at the begging of the new line. 
If you take a look at standard processors, all use the other format.

Reply:
Done in new commit.